qemu/tests/cfg/cgroup.cfg

```diff
@@ -96,5 +96,5 @@
         - cfs_bandwidth:
             start_vm = "no"
             check_install_stress = "which stress"
-            stress_cmd = "stress -c 1 -t 100"
+            stress_cmd = "stress -c 4 -t 200"
```

Comment:
values of -c "option" should match with  vcpu counts but not static value in here, right?

Reply:
400%  overcommit maybe caused your  system hang

Reply:
updated,  As vcpu number is 1 for this case. so use '-c 1'  (static) option.